cdap-app-fabric/src/main/java/co/cask/cdap/app/guice/AppFabricServiceRuntimeModule.java

```diff
@@ -318,6 +320,7 @@ protected void configure() {
       bind(RuntimeStore.class).to(DefaultStore.class);
       bind(ArtifactStore.class).in(Scopes.SINGLETON);
       bind(ProgramLifecycleService.class).in(Scopes.SINGLETON);
+      bind(PreviewStore.class).to(DefaultPreviewStore.class);
```

Comment:
Why add to this module? Shouldn't it in the preview module only? Adding it here means it will affect non-preview JVM as well.